packages/rocketchat-file-upload/server/methods/sendFileMessage.js

```diff
@@ -32,42 +32,46 @@ Meteor.methods({
 			title_link_download: true
 		};
 
-		if (/^image\/.+/.test(file.type)) {
-			attachment.image_url = fileUrl;
-			attachment.image_type = file.type;
-			attachment.image_size = file.size;
-			if (file.identify && file.identify.size) {
-				attachment.image_dimensions = file.identify.size;
+		Meteor.wrapAsync(FileUpload.resizeImagePreview(file, Meteor.bindEnvironment(function(r) {
```

Comment:
Are you resizing even not image files?

Reply:
Sorry, I forget to put a test at the function that makes the resize, I will fix this too

Reply:
I added to the file that test, but i'm can not fixing the build error, the log says that are a meteor promise module error.